src/pages/news/2024-04-15-a-tale-of-two-bugs.md

```diff
@@ -1,17 +1,26 @@
 ---
 template: news-item
-title: A tale of two bugs
-description: The FTX exploit attempted redemption of tBTC revealed two bugs.
+title: "A Tale of Two Bugs: Uncovering Vulnerabilities in tBTC Amid the FTX Exploit"
+description: Explore how the FTX exploit uncovered two critical bugs in tBTC, including a denial-of-service vector and a redemption mechanism design flaw, and the steps taken to address them.
 canonicalUrl: https://blog.threshold.network/a-tale-of-two-bugs/
 date: 2024-04-15T12:17:09.130Z
+tags:
+  - featured
+  - tBTC
+  - FTX exploit
+  - Bitcoin bridge
+  - DeFi security
 ---
+## Examining the Impact of Two Bugs Exposed by the FTX Exploit
+
+
 As has been reported this week, an address associated with the FTX exploit has been moving funds through a number of cross-chain projects.
 
 While most of the funds have [gone through Thorchain](https://www.theblock.co/post/255108/thorswap-dex-enters-maintenance-mode-amid-illicit-activity?ref=blog.threshold.network), some of them have been routed through tBTC. In the process, two bugs have been exposed.
 
 Neither bug puts user funds at risk. The first was patched and released yesterday, while the second requires community discussion and consensus.
 
-## The first bug — a denial-of-service vector
+### The First Bug: A Denial-of-Service Vector
```

Comment:
Is there a reason to editorialize all of these titles, or do you just hate my originals? :stuck_out_tongue:

Reply:
We shouldn't modify at all here, this is a replica.

Reply:
> Is there a reason to editorialize all of these titles, or do you just hate my originals? 😛

truly? i just hate it 🤣 i think we should standardize titles and subtitle letter case across the board. it's neater.

Reply:
Not on cross-posts, not today :)